<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>RapidSum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="chkFullPath">
      <property name="text">
       <string>Display the full path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cmbHashType">
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>MD5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SHA-1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CRC32</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnAbout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Checksum</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Files"/>
   <addaction name="actionAdd_Folder"/>
   <addaction name="actionRemove"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>3</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <action name="actionAdd_Files">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/addfile.png</normaloff>:/images/addfile.png</iconset>
   </property>
   <property name="text">
    <string>Add File(s)</string>
   </property>
  </action>
  <action name="actionAdd_Folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/addfolder.png</normaloff>:/images/addfolder.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="toolTip">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
